website/docs/r/lex_bot.html.markdown

```diff
@@ -0,0 +1,190 @@
+---
+layout: "aws"
+page_title: "AWS: aws_lex_bot"
+sidebar_current: "docs-aws-resource-lex-bot"
+description: |-
+  Provides an Amazon Lex bot resource.
+---
+
+# aws_lex_bot
+
+Provides an Amazon Lex Bot resource. For more information see
+[Amazon Lex: How It Works](https://docs.aws.amazon.com/lex/latest/dg/how-it-works.html)
+
+## Example Usage
+
+```hcl
+resource "aws_lex_bot" "order_flowers_bot" {
+  abort_statement {
+    message {
+      content      = "Sorry, I am not able to assist at this time"
+      content_type = "PlainText"
+    }
+  }
+
+  child_directed = false
+
+  clarification_prompt {
+    max_attempts = 2
+
+    message {
+      content      = "I didn't understand you, what would you like to do?"
+      content_type = "PlainText"
+    }
+  }
+
+  description = "Bot to order flowers on the behalf of a user"
+
+  idle_session_ttl_in_seconds = 600
+
+  intent {
+    intent_name    = "OrderFlowers"
+    intent_version = "1"
+  }
+
+  locale           = "en-US"
+  name             = "OrderFlowers"
+  process_behavior = "BUILD"
+  voice_id         = "Salli"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+### Required
+
+* `abort_statement`
+
+	The message that Amazon Lex uses to abort a conversation.
+
+    * Type: [Lex Statement](/docs/providers/aws/r/lex_statement.html)
+
+* `child_directed`
+
+	By specifying true, you confirm that your use of Amazon Lex is related to a website, program,
+	or other application that is directed or targeted, in whole or in part, to children under age
+	13 and subject to COPPA.
+
+	By specifying false, you confirm that your use of Amazon Lex is not related to a website,
+	program, or other application that is directed or targeted, in whole or in part, to children
+	under age 13 and subject to COPPA.
+
+    If your use of Amazon Lex relates to a website, program, or other application that is directed
+	in whole or in part, to children under age 13, you must obtain any required verifiable
+	parental consent under COPPA. For information regarding the use of Amazon Lex in connection
+	with websites, programs, or other applications that are directed or targeted, in whole or in
+	part, to children under age 13, see the
+	[Amazon Lex FAQ](https://aws.amazon.com/lex/faqs#data-security).
+
+    * Type: bool
+
+* `clarification_prompt`
+
+	The message that Amazon Lex uses when it doesn't understand the user's request.
+
+    * Type: [Lex Prompt](/docs/providers/aws/r/lex_prompt.html)
+
+* `intent`
+
+	A set of Intent objects. Each intent represents a command that a user can express.
+
+    * Type: Set of [Lex Bot Intents](/docs/providers/aws/r/lex_bot_intent.html)
+
+* `name`
+
+	The name of the bot that you want to create a new version of. The name is case sensitive.
```

Comment:
Maybe I just don't understand what this does but I'm confused by the words `a new version of`, how about `The name of the bot that you want to create, case sensitive.`?

Reply:
Correct. Typically I was just copying these descriptions from the API reference. Not sure what happened here.

```suggestion
	The name of the bot that you want to create, case sensitive.
```